packages/lightcone_v2.js/src/lib/wallet/ethereum/walletAccount.ts

```diff
@@ -193,8 +193,10 @@ export class WalletAccount {
   //     throw Error('unimplemented');
   // }
 
-  sendTransaction(ethNode, signedTx) {
-    return ethNode.sendRawTransaction(signedTx);
```

Comment:
If we don't catch response of sendRawTransation, downstream request won't be able to catch the error. Then it will fail silently.

We don't have to use await and async here. Promise also works. If we use Promise, we will need resolve or reject to catch the response.